projects/layout/components/dynamic-header/dynamic-header.animations.ts

```diff
@@ -0,0 +1,26 @@
+import {animate, style, transition, trigger} from '@angular/animations';
+
+export const tuiSlideInTopHeader = trigger('tuiSlideInTopHeader', [
```

Comment:
Could we somehow reuse already existing `@tuiSlideIn...` animations ?

https://taiga-ui.dev/animations#tui-slide-in

Reply:
Unfortunately no, we need different params for leave and enter animations. There is no way to reuse this without breaking changes